From 4c22c9922e81904410d8f97065a267dee7b99380 Mon Sep 17 00:00:00 2001
From: Peter Chen <peter.chen@freescale.com>
Date: Tue, 14 Aug 2012 15:37:42 +0800
Subject: [PATCH 1914/2463] ENGR00220341-3 usb: add spin_lock at platform data

It is used to sync pdata->lowpower between wakeup interrupt
and driver API.

Signed-off-by: Peter Chen <peter.chen@freescale.com>
---
 include/linux/fsl_devices.h |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/include/linux/fsl_devices.h b/include/linux/fsl_devices.h
index 8c41b23..2a95573 100644
--- a/include/linux/fsl_devices.h
+++ b/include/linux/fsl_devices.h
@@ -144,6 +144,7 @@ struct fsl_usb2_platform_data {
 	unsigned	irq_delay:1;
 	enum usb_wakeup_event	wakeup_event;
 	u32		pmflags;	/* PM from otg or system */
+	spinlock_t lock;
 
 	void __iomem *charger_base_addr; /* used for i.mx6 usb charger detect */
 
-- 
1.7.7.4

